.ci/buildchain-project-dependencies.yaml

```diff
@@ -8,8 +8,12 @@ dependencies:
       dependencies:
         default:
           - source: development
+            target: main # should migrate to 9 before the build
+          - source: 8.x # does not migrate
             target: main
       dependant:
         default:
-          - source: main
+          - source: 9.x
+            target: development
+          - source: main # should migrate to 9 before the build
```

Comment:
so from optaplanner:main, we don't check quickstarts:8.x ?

Reply:
It's the question of who is the first-class citizen in the quickstarts repo. In my opinion, it's the `development` branch, which contains the latest version.
The `8.x` is there for the next product release and will not receive any updates apart from necessary fixes.

Thus, I would like to check if the `development` branch is impacted by any new PR to the `optaplanner/main`. For `8.x`, nightly or a custom GHA after merging is enough.

On the other hand, if the CI is triggered on the `quickstarts/8.x`, I want the `optaplanner/main` without migration to be its upstream.

That's the ideal situation; whether it's realistic or not, I don't know.